sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -785,6 +785,13 @@ object SQLConf {
     .booleanConf
     .createWithDefault(false)
 
+  val PARQUET_META_CACHE_ENABLED = buildConf("spark.sql.parquet.metadataCache.enabled")
```

Comment:
Do we have another configuration in `spark.sql.parquet.metadataCache.` namespace? If not, we usually do not make a new namespace.